src/main/java/it/pagopa/ecommerce/commons/domain/v1/TransactionWithRefundError.java

```diff
@@ -40,10 +41,10 @@ public final class TransactionWithRefundError extends BaseTransactionWithComplet
      * @param transactionRefundErrorEvent transaction refund error event
      */
     public TransactionWithRefundError(
-            BaseTransactionWithCompletedAuthorization baseTransaction,
+            BaseTransaction baseTransaction,
```

Comment:
This can be constructed from any transaction, while it should be possible only in a transaction in a `REFUND_REQUESTED` state (`BaseTransactionWithRefundRequested`?)

Reply:
Fixed with https://github.com/pagopa/pagopa-ecommerce-commons/pull/48/commits/ee1b7b011bdfd964c59b27821ac2551f01cd359f